components/metal/BuildingSection.vue

```diff
@@ -0,0 +1,136 @@
+<template>
+  <section class="building-section">
```

Comment:
Building Section refers to a section that will be only in one page, not once per page, so, according the [Nuxt.js conventions](https://vuejs.org/v2/style-guide/#Single-instance-component-names-strongly-recommended): "This does not mean the component is only used in a single page, but it will only be used once per page.", I think we shouldn't add the `The` prefix. Same for the following sections (CapabilitiesSection, EarlyAccessSection, FeaturesSection👇

Reply:
According to the convention:
> Components that should only ever have a single active instance should begin with the `The` prefix, to denote that there can be only one.

It is my understanding that all the components under the page fall in this category. There will be only one active instance of each of them. So they should be prefixed with `The`.

The clarification says that the previous statement does not mean it must be just once per site, but once per page.

This is the same we did in other parts of the site such as [the index](https://github.com/Qiskit/qiskit.org/blob/master/pages/index.vue) or [the learn index](https://github.com/Qiskit/qiskit.org/blob/master/pages/learn/index.vue).

Reply:
As I see it, there are two types of components that could be used only once on pages:
- A component that is used across all the web, generally included as part of the main layout
- A component that, at the moment, is used only once in one place

I think this style recommendation refers to the first one, a component used across all the web, only once per page, for example: `TheMenu`, `TheFooter`.

I know that we have done that in other pages for other components used only once on one page, and I still think this is not the proper way to do it.

In my opinion, the only components that should have the `The` prefix are the ones that are used across the web always the same way, i.e., they are set on the general layouts

In my opinion, the only components that should have the `The` prefix are the ones that are used accross the web always the same way, i.e, they are set on the general layouts

Reply:
So if these elements do not fall under the category that qualifies for being prefixed, I assume that you foresee being reused at some moment in the future. I think we will never reuse them. Perhaps there is a third category for those.

Anyway, this exposes an ambiguity in our code style guides and potential technical debt. Regardless what you decide to do, please open a discussion issue to clarify the rule a get an agreement with the rest of the team, like we did in #660. If you decide not to modify the PR, also open an issue to track the fact that some other components should adhere to your interpretation.